### PR TITLE
Further basis for themes auto-updates

### PR DESCRIPTION
**Pull request by bookdude13**
Hey @audrasjb,

I had some time on my hands and found some of the JS hacks needed to get the updates with themes to work. This is built on top of the existing basis from #44. Let me know if there is a cleaner solution; right now the template is modified through some string replacement using jQuery. I haven't verified that the toggles actually work to enable/disable the updates, but the general logic flow works.

I wanted to post this now to avoid duplicate work being done 😃

### CODE REVIEW

**Comment by bookdude13** [comment]
Debug information now added, and this appears to be working for me on single site from themes.php. Cleanup for setting theme values not possible yet - see https://core.trac.wordpress.org/ticket/14955.

File: wp-autoupdates.php [hunk 1]
```diff
@@ -61,10 +61,87 @@ function wp_autoupdates_enqueues( $hook ) {
...
+			$script = 'jQuery( document ).ready( function() {';
+
+			/* translators: %s: Theme name. */
+			$aria_label_enable  = sprintf( _x( 'Enable automatic update for %s', 'theme' ), '{{ data.name }}' );
+ 			$aria_label_disable = sprintf( _x( 'Disable automatic update for %s', 'theme' ), '{{ data.name }}' );
+
+			// Put the enable/disable link below the author and before the update box.
+			$autoupdate_text = '<p class="theme-autoupdate"> <# if ( data.autoupdate ) { #>';
+			$autoupdate_text .= '<span class="theme-autoupdate-disabled">';
+			$autoupdate_text .= '<a href="{{{ data.actions.autoupdate }}}" aria-label="' . $aria_label_disable . '"><span class="dashicons dashicons-update" aria-hidden="true"></span>' . __( 'Disable automatic updates' ) . '</a>';
```
Comment (audrasjb) [inline wp-autoupdates.php]: Would be nice to add a space after the dashicon’s `<span>` element :)

File: wp-autoupdates.php [hunk 2]
```diff
@@ -61,10 +61,87 @@ function wp_autoupdates_enqueues( $hook ) {
...
+			$autoupdate_text .= '</span>';
+			$autoupdate_text .= '<# } else { #>';
+			$autoupdate_text .= '<span class="theme-autoupdate-enabled">';
+			$autoupdate_text .= '<a href="{{{ data.actions.autoupdate }}}" aria-label="' . $aria_label_enable . '"><span class="dashicons dashicons-update" aria-hidden="true"></span>' . __( 'Enable automatic updates' ) . '</a>';
```
Comment (audrasjb) [inline wp-autoupdates.php]: Would be nice to add a space after the dashicon’s `<span>` element as well :)

**Review by audrasjb — CHANGES_REQUESTED** [review]
Thanks @bookdude13 ! Looks great to me :)
Just added two small items to fix before merge :)

**Comment by bookdude13** [comment]
Glad to help! Spaces added, should be good to go 😃

**Review by audrasjb — APPROVED** [review]

**Pull request merged**